### ISSUE TITLE
the keys command only returns of the current node

### ISSUE THREAD

**Issue by whimthen** [laixintao/iredis#363]
In cluster mode, the `keys` command only returns the matching key of the current node, and does not return the keys of other nodes

<img width="388" alt="image" src="https://user-images.githubusercontent.com/26761592/98782397-df678400-2432-11eb-8192-b3f87cec9a1f.png">

<img width="262" alt="image" src="https://user-images.githubusercontent.com/26761592/98782443-ee4e3680-2432-11eb-916d-b9a9e1d90ff3.png">

**Comment by laixintao** [comment]
Yes, `KEYS` and `SCAN` are for the currently connected node, I can't come up with a good way of querying all keys among clusters. Do you have any ideas?

And I don't think redis-cli support this either.

**Comment by whimthen** [comment]
redis-cli is not supported, but it would be better if there is a way to query the experience of all nodes.😀😀

**Comment by laixintao** [comment]
Good idea, I think we can combine with other commands to implement this
feature :)
